scripts/ci/openapi/client_codegen_diff.py

```diff
@@ -0,0 +1,59 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+from __future__ import annotations
+
+import subprocess
+
+
+def run_command(commands: list):
+    result = subprocess.run(commands, check=True, capture_output=True, text=True)
+    if result.returncode != 0:
+        print(f"Error running command: {result.stderr}")
+        exit(1)
+    return result.stdout
+
+
+# HEAD^1 says the "first" parent. For PR merge commits, or main commits, this is the "right" commit.
+#
+# In this example, 9c532b6 is the PR commit (HEAD^2), 4840892 is the head GitHub checks-out for us,
+# and db121f7 is the "merge target" (HEAD^1) -- i.e. mainline
+
+def client_codegen_diff():
```

Comment:
As discussed on slack. I think this script could be simply converted in the `breeze ci build-api-client` command.

There are multiple benefits (from ease of use starting, through autocomplete and auatomated documenation, ending with automatically managed virtualenv).

Also I think that could be useful to use Python native stdlib where needed - for example `mkdir` could be easily done using pathlib

Reply:
I will convert it into a breeze command
Is there any existing file that is suitable for this command or should it be a new one?

Reply:
I think it fits best into `release-management` command group.